lib/client/api.go

```diff
@@ -281,6 +281,10 @@ type Config struct {
 	// if omitted, first available site will be selected
 	SiteName string
 
+	// ExplicitSiteNameis is true if SiteName was initially set by the end-user
+	// (for example, using command-line flags).
+	ExplicitSiteName bool
```

Comment:
Is this used anywhere? I see it's being set in a couple of places but couldn't find if it's actually being used by anything. If it is - what for?